### PR TITLE
some helper functions for Expand Article tests

### PR DESCRIPTION
**Pull request by Jenniferstrej**
one folder wasn't being deleted properly

### CODE REVIEW

File: tests/activity/test_activity_expand_article.py [hunk 1]
```diff
@@ -87,10 +89,10 @@ def test_do_activity_invalid_status(self, status_example, mock_storage_context,
         self.assertEqual(self.expandarticle.ACTIVITY_PERMANENT_FAILURE, success)
 
 
-    def delete_files_in_folder(self, folder):
-        fileList = os.listdir(folder)
-        for fileName in fileList:
-            os.remove(folder+"/"+fileName)
+    # def delete_files_in_folder(self, folder):
```
Comment (giorgiosironi) [inline tests/activity/test_activity_expand_article.py]: think you can now delete this commented out code?

**Comment by giorgiosironi** [comment]
:+1: 
by the way do we have any convention for variable and function names? `snake_case` or `camelCase` for which of them?

**Comment by gnott** [comment]
Name conventions, I think we're going for underscores for variables in general. Though, because there's some creative legacy names in elife-bot right now, I didn't want to enforce naming checks.

**Comment by giorgiosironi** [comment]
Ok, so in the new code being added here things like `fileName` and `fileList` should be `file_name` and `file_list` and so on.

**Comment by Jenniferstrej** [comment]
Thanks for the code review. Will tidy up... :)

On 22 June 2016 at 16:38, Giorgio Sironi <notifications@github.com> wrote:

>
> <https://camo.githubusercontent.com/d00625d32253cf7c9ffdc6007cf5d5b676d0cfe0/687474703a2f2f696d6167652e736c696465736861726563646e2e636f6d2f32303135746563686e6963616c646562742d3135303431313132303132372d636f6e76657273696f6e2d6761746530312f39352f7079636f6e2d323031352d746563686e6963616c2d646562742d7468652d6d6f6e737465722d696e2d796f75722d636c6f7365742d37312d3633382e6a70673f63623d31343239353238323035>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/elifesciences/elife-bot/pull/255#issuecomment-227784258>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AC4pPvxef3-Vw_WbUcqLWKydIhZXfGtpks5qOVb1gaJpZM4I731D>
> .
>



-- 
Jennifer

**Comment by giorgiosironi** [comment]
Posthumous :+1:

**Pull request merged**